redis/redispipebp/BUILD.bazel

```diff
@@ -0,0 +1,56 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "baseplate.go",
+        "breaker.go",
+        "config.go",
+        "doc.go",
+        "errors.go",
+        "monitored.go",
+        "retry.go",
+    ],
+    importpath = "github.com/reddit/baseplate.go/redis/redispipebp",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//breakerbp:go_default_library",
+        "//redis/redisx:go_default_library",
+        "//retrybp:go_default_library",
+        "//tracing:go_default_library",
+        "@com_github_avast_retry_go//:go_default_library",
+        "@com_github_joomcode_errorx//:go_default_library",
+        "@com_github_joomcode_redispipe//redis:go_default_library",
+        "@com_github_joomcode_redispipe//rediscluster:go_default_library",
+        "@com_github_joomcode_redispipe//redisconn:go_default_library",
+        "@com_github_opentracing_opentracing_go//:go_default_library",
+    ],
+)
+
+go_test(
+    name = "go_default_test",
+    srcs = [
```

Comment:
nit: add `size = "small",` between the `name` and `srcs` lines.

Reply:
No problem but what does that do?

Reply:
without it bazel will print out a warning when running with `--test_verbose_timeout_warnings` arg, and we enforced that arg in [our `.bazelrc` file](https://github.com/reddit/baseplate.go/blob/master/.bazelrc).

basically bazel test uses the size (or similar tags) info to decide what the timeout for this test should be (larger tests will have longer timeouts before declare failure), and this info also helps bazel to figure out a strategy/policy to run tests in parallel, etc.